src/datetime/mod.rs

```diff
@@ -848,13 +848,24 @@ impl DateTime<Utc> {
     /// let timestamp_nanos: i64 = -2208936075_000_000_000; // Mon, 1 Jan 1900 14:38:45 UTC
```

Comment:
We have a document that this method would never fail in line 835

Reply:
This method would never fail, added doc tests for `from_timestamp_nanos` with parameter `i64::MIN` and `i64::MAX`, which aims to show this more clearly